content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

```diff
@@ -1,76 +1,199 @@
+For example with an HPA query, the metrics-server needs to identify which pods fulfill the label selectors in the deployment.
 
-Learn more about the metrics server in
-[the design doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/metrics-server.md).
+The metrics-server employs the Summary API to fetch metrics exposed by
+[kubelet](/docs/reference/command-line-tools-reference/kubelet/) on each node.
 
-### Summary API Source
-The [Kubelet](/docs/reference/command-line-tools-reference/kubelet/) gathers stats at node, volume, pod and container level, and emits their statistics in
+To learn more about the metrics-server, see the [metrics-server repository](https://github.com/kubernetes-sigs/metrics-server).
+
+You can also check out the following:
+
+* [metrics-server design](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/metrics-server.md)
+* [metrics-server FAQ](https://github.com/kubernetes-sigs/metrics-server/blob/master/FAQ.md)
+* [metrics-server known issues](https://github.com/kubernetes-sigs/metrics-server/blob/master/KNOWN_ISSUES.md)
+* [metrics-server releases](https://github.com/kubernetes-sigs/metrics-server/releases)
+* [Horizontal Pod Autoscaling](/docs/tasks/run-application/horizontal-pod-autoscale/)
+
+### Summary API source
+
+The [kubelet](/docs/reference/command-line-tools-reference/kubelet/) gathers stats at the node, volume, pod and container level, and emits this information in
 the [Summary API](https://github.com/kubernetes/kubernetes/blob/7d309e0104fedb57280b261e5677d919cb2a0e2d/staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go)
 for consumers to read.
 
-Pre-1.23, these resources have been primarily gathered from [cAdvisor](https://github.com/google/cadvisor). However, in 1.23 with the
-introduction of the `PodAndContainerStatsFromCRI` FeatureGate, container and pod level stats can be gathered by the CRI implementation.
-Note: this also requires support from the CRI implementations (containerd >= 1.6.0, CRI-O >= 1.23.0).
+Here is an example of a Summary API request for a `minikube` node:
+```text
+kubectl get --raw /api/v1/nodes/minikube/proxy/stats/summary
+```
+Here is the same API call using `curl`:
+```text
+curl http://localhost:8080/api/v1/nodes/minikube/proxy/stats/summary
+```
+
+{{< note >}}
+Metrics-server v0.6.0 replaces the summary API `/nodes/stats/` endpoint with the kubelet API `/nodes/metrics/resource` endpoint for pulling stats. For more details, see [metrics-server releases](https://github.com/kubernetes-sigs/metrics-server/releases).
+{{< /note >}}
+
+Pre-K8s 1.23, these resources have been primarily gathered from [cAdvisor](https://github.com/google/cadvisor). However, in 1.23 with the introduction of the `PodAndContainerStatsFromCRI` FeatureGate, container and pod level stats can be gathered by the CRI implementation. 
```

Comment:
Not sure if this is correct. I think the pre 1.23 behavior was based on fallback. cAdvisor -> CRI. I expect that this feature gate changed the preference to pick CRI over cAdvisor.